portal-impl/src/com/liferay/portal/language/LanguageImpl.java

```diff
@@ -2017,7 +2017,7 @@ private void _updateLastModified() {
 		LanguageImpl.class.getName() + "._groupLocalesPortalCache";
 
 	private static final String _LIFERAY_LANGUAGE_IMPORT_REGEXP =
-		"await import\\(.@liferay/language/.+/all\\.js.\\)";
+		"await import\\(.@liferay/language/.+?/all\\.js.\\)";
```

Comment:
Just for the record: I've double tested the regexp at https://www.freeformatter.com/java-regex-tester.html and it seems correct. :muscle: